scripts/build.py

```diff
@@ -1101,6 +1125,23 @@ def generate_build_directory():
     print 'Build completed.'
 
 
+def compile_typescript_files(
+        file, outDir='backend_prod_files', allowJS='true',
```

Comment:
Python varnames use snake_case.

Also, if you are not actually passing in different args to override, it's fine to remove those from the arg list and inline them instead. (Though you might want to pass in out_dir explicitly, but in that case don't include a default arg.)

Reply:
Done.